apps/evans/justfile

```diff
@@ -1,18 +1,21 @@
 _default:
   @just --choose
 
-dev-en: write-cover-css
+dev-en: lang-en write-cover-css
   @NEXT_PUBLIC_LANG=en pnpm next dev --port 7222
 
-dev-es: write-cover-css
+dev-es: lang-es write-cover-css
   @NEXT_PUBLIC_LANG=es pnpm next dev --port 7333
 
-build-en: write-cover-css
+build-en: lang-en write-cover-css web-covers && unweb-covers
   @NEXT_PUBLIC_LANG=en pnpm next build
 
-build-es: write-cover-css
+build-es: lang-es write-cover-css web-covers && unweb-covers
   @NEXT_PUBLIC_LANG=es pnpm next build
 
+build-analyze:
+  @ANALYZE=true just build-en
```

Comment:
i'll show you more about this in the future, but this command allows us to generate a bundle report (similar to this one from our old gatsby site:  https://flp-assets.nyc3.digitaloceanspaces.com/actions/bundle/master.html

i used it to find a bunch of low-hanging fruit, and so that i could measure and see real differences from what i was working on.

there are more applications and things i want to do with this going forward, but i'll save that for future PR's and possible pair sessions.